docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -310,13 +310,67 @@ You can publish collections to Galaxy using the ``ansible-galaxy collection publ
 Getting your token or API key
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To upload your collection to Galaxy, you must first obtain an API token (``--api-key`` in the ``ansible-galaxy`` CLI command). The API token is a secret token used to protect your content.
+To upload your collection to Galaxy, you must first obtain an API token (``--api-key`` in the ``ansible-galaxy`` CLI command or ``token`` in the :file:`ansible.cfg` file under the ``galaxy`` section). The API token is a secret token used to protect your content.
 
 To get your API token:
 
 * For galaxy, go to the `Galaxy profile preferences <https://galaxy.ansible.com/me/preferences>`_ page and click :guilabel:`API token`.
 * For Automation Hub, go to https://cloud.redhat.com/ansible/automation-hub/token/ and click :guilabel:`Get API token` from the version dropdown.
 
+Storing or using your API token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have retrieved your AP token, you have three options on how you can store or use that token for collections:
+
+* Pass the token to  the ``ansible-galaxy`` command using the ``--api-key``.
+* Save the token in a Galaxy token file.
```

Comment:
I'm a bit hesitant to document this, if `ansible-galaxy login` ever goes away then this would probably go with it. IMO we should only have the `--api-key` and entries in `ansible.cfg`.

Reply:
removed.